*** 2017-06-19 20:39:31.079 idempiere Log (CLogFile) ***
20:39:31.079 Adempiere.startup: iDempiere(r) Release 3.1_2015-10-31 -Smart Suite ERP, CRM and SCM- (c) 1999-2013 iDempiere(r); Implementation: iDempiere - Supported by iDempiere community [17]
20:39:31.081 Ini.loadProperties: E:\mercurialidempiere/idempiereDEMO.properties #33 [17]
20:39:36.596 AdempiereActivator.installPackage: ni.hmil.override.codebase 1.0.0 was installed: 2017-06-18 10:07:05.297 [17]
20:39:39.591 DB.isBuildOK: Build DB=${env.ADEMPIERE_VERSION} 20080428-1232 [17]
20:39:39.592 DB.isBuildOK: Build Cl=3.1.0.qualifier [17]
20:39:41.605-----------> DB.isBuildOK: Error en la versión de la construcción 

El programa asume version de construcción 3.1.0.qualifier, pero la base de datos tiene una versión ${env.ADEMPIERE_VERSION} 20080428-1232.
Es posible que esto cause errores difíciles de corregir.
Por favor contacte al ADMINISTRADOR. [17]
20:39:48.244 WebEnv.initWeb: Servlet Context Init Parameters: iDempiere Root
ServerRoot=1 [17]
20:39:48.467-----------> EMail.setFrom: you @ company.org: javax.mail.internet.AddressException: Local address contains control or whitespace in string ``you @ company.org'' [17]
20:39:49.111-----------> EMail.addTo: you @ company.org: javax.mail.internet.AddressException: Local address contains control or whitespace in string ``you @ company.org'' [17]
20:39:49.111-----------> EMail.isValid: From is invalid=null [17]
20:39:49.111-----------> EMail.isValid: From is invalid=null [17]
20:39:49.111-----------> MClient.sendEMail: Could NOT Send Email: Server started: iDempiere (odbz:192.168.47.1) to you @ company.org: Invalid Data (System) [17]
20:39:49.111 WebEnv.initWeb: Servlet Init Parameter: idempiereMonitor [17]
20:39:49.111 AdempiereMonitor.init:  [17]
20:40:39.691-----------> AnnotationConfiguration.configure: ServletContainerInitializers: detected. Class hierarchy: empty [17]
20:40:39.904-----------> ClassLocator.getDependentXMLResources: Replicate resource: ckez
Overwrite bundleresource://182.fwk1480207683:2/metainfo/zk/config.xml
with bundleresource://182.fwk1480207683:19/metainfo/zk/config.xml [17]
20:40:39.909-----------> ClassLocator.getDependentXMLResources: Replicate resource: zk
Overwrite bundleresource://182.fwk1480207683:11/metainfo/zk/config.xml
with bundleresource://182.fwk1480207683:28/metainfo/zk/config.xml [17]
20:40:39.914-----------> ClassLocator.getDependentXMLResources: Replicate resource: zkbind
Overwrite bundleresource://182.fwk1480207683:12/metainfo/zk/config.xml
with bundleresource://182.fwk1480207683:29/metainfo/zk/config.xml [17]
20:40:39.936-----------> ClassLocator.getDependentXMLResources: Replicate resource: zul
Overwrite bundleresource://182.fwk1480207683:14/metainfo/zk/zk.xml
with bundleresource://182.fwk1480207683:31/metainfo/zk/zk.xml [17]
20:40:40.961-----------> LanguageDefinition.<init>: Replicated language: xhtml, overriden by [LanguageDefinition: xhtml] [17]
20:40:40.991-----------> LanguageDefinition.<init>: Replicated language: xul/html, overriden by [LanguageDefinition: xul/html] [17]
20:40:41.045-----------> ClassLocator.getDependentXMLResources: Replicate resource: calendar
Overwrite bundleresource://182.fwk1480207683:1/metainfo/zk/lang-addon.xml
with bundleresource://182.fwk1480207683:18/metainfo/zk/lang-addon.xml [17]
20:40:41.049-----------> ClassLocator.getDependentXMLResources: Replicate resource: ckez
Overwrite bundleresource://182.fwk1480207683:2/metainfo/zk/lang-addon.xml
with bundleresource://182.fwk1480207683:19/metainfo/zk/lang-addon.xml [17]
20:40:41.056-----------> ClassLocator.getDependentXMLResources: Replicate resource: gmapsz
Overwrite bundleresource://182.fwk1480207683:5/metainfo/zk/lang-addon.xml
with bundleresource://182.fwk1480207683:22/metainfo/zk/lang-addon.xml [17]
20:40:41.060-----------> ClassLocator.getDependentXMLResources: Replicate resource: timeline
Overwrite bundleresource://182.fwk1480207683:6/metainfo/zk/lang-addon.xml
with bundleresource://182.fwk1480207683:23/metainfo/zk/lang-addon.xml [17]
20:40:41.066-----------> ClassLocator.getDependentXMLResources: Replicate resource: timeplot
Overwrite bundleresource://182.fwk1480207683:7/metainfo/zk/lang-addon.xml
with bundleresource://182.fwk1480207683:24/metainfo/zk/lang-addon.xml [17]
20:40:41.090-----------> ClassLocator.getDependentXMLResources: Replicate resource: zkbind
Overwrite bundleresource://182.fwk1480207683:12/metainfo/zk/lang-addon.xml
with bundleresource://182.fwk1480207683:29/metainfo/zk/lang-addon.xml [17]
20:40:41.101-----------> ClassLocator.getDependentXMLResources: Replicate resource: zkplus
Overwrite bundleresource://182.fwk1480207683:13/metainfo/zk/lang-addon.xml
with bundleresource://182.fwk1480207683:30/metainfo/zk/lang-addon.xml [17]
20:40:42.011           WebUIServlet.init: iDempiere Web Client started successfully [17]
20:40:43.366-----------> AnnotationConfiguration.configure: ServletContainerInitializers: detected. Class hierarchy: empty [17]
20:40:43.435-----------> Prop.loadProperties: E:\mercurialidempiere\home.properties not found [35]
20:40:43.435-----------> Prop.loadProperties: !!WARNING:Please locate yout custom home.properties on iDEMPIERE_HOME. [35]
20:40:43.514-----------> LCO_ValidatorWH.initialize:  [48]

*** 2017-06-19 20:40:43.735 idempiere Log (CLogFile) ***
20:40:46.483-----------> Node: [192.168.163.1]:5701 [adempiere.idempiere/localhost] [3.5.3] No join method is enabled! Starting standalone. [50]
20:46:15.381-----------> Msg.getMsg: NOT found: Do you realy want to erase queue ? [64]
20:46:49.548===========> TableIndexElementHandler.validateTableIndex: ERROR: column am.amcanorder does not exist
  Position: 427 [41]
org.postgresql.util.PSQLException: ERROR: column am.amcanorder does not exist
  Position: 427; State=42703; ErrorCode=0
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2182)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1911)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:481)
	at org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:361)
	at org.postgresql.jdbc2.AbstractJdbc2DatabaseMetaData.getIndexInfo(AbstractJdbc2DatabaseMetaData.java:4230)
	at com.mchange.v2.c3p0.impl.NewProxyDatabaseMetaData.getIndexInfo(NewProxyDatabaseMetaData.java:3328)
	at org.compiere.process.TableIndexValidate.validateTableIndex(TableIndexValidate.java:69)
	at org.adempiere.pipo2.handler.TableIndexElementHandler.validateTableIndex(TableIndexElementHandler.java:112)
	at org.adempiere.pipo2.handler.TableIndexElementHandler.endElement(TableIndexElementHandler.java:93)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:266)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:258)
	at org.adempiere.pipo2.PackInHandler.endElement(PackInHandler.java:334)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.endElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEndElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)

20:47:00.809===========> PackIn.importXML: importXML: [41]
org.adempiere.pipo2.exception.DatabaseAccessException: Failed to validate AD_TableIndex for lco_invoicewithholding_uu_idx
	at org.adempiere.pipo2.handler.TableIndexElementHandler.endElement(TableIndexElementHandler.java:99)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:266)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:258)
	at org.adempiere.pipo2.PackInHandler.endElement(PackInHandler.java:334)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.endElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEndElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse(Unknown Source)
	at javax.xml.parsers.SAXParser.parse(Unknown Source)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:173)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:136)
	at org.adempiere.pipo.srv.PipoDictionaryService.merge(PipoDictionaryService.java:104)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:198)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:166)

20:47:05.319===========> PackIn.importXML: importXML: [41]
java.lang.RuntimeException: Failed to validate AD_TableIndex for lco_invoicewithholding_uu_idx
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:188)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:136)
	at org.adempiere.pipo.srv.PipoDictionaryService.merge(PipoDictionaryService.java:104)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:198)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:166)
	at org.adempiere.plugin.utils.Incremental2PackActivator.installPackage(Incremental2PackActivator.java:120)
	at org.adempiere.plugin.utils.Incremental2PackActivator.access$4(Incremental2PackActivator.java:89)
	at org.adempiere.plugin.utils.Incremental2PackActivator$2.run(Incremental2PackActivator.java:259)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
caused by: org.adempiere.pipo2.exception.DatabaseAccessException: Failed to validate AD_TableIndex for lco_invoicewithholding_uu_idx
	at org.adempiere.pipo2.handler.TableIndexElementHandler.endElement(TableIndexElementHandler.java:99)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:266)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:258)
	at org.adempiere.pipo2.PackInHandler.endElement(PackInHandler.java:334)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.endElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEndElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse(Unknown Source)
	at javax.xml.parsers.SAXParser.parse(Unknown Source)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:173)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:136)
	at org.adempiere.pipo.srv.PipoDictionaryService.merge(PipoDictionaryService.java:104)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:198)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:166)

20:47:09.726===========> PipoDictionaryService.merge: importXML: [41]
java.lang.RuntimeException: Failed to validate AD_TableIndex for lco_invoicewithholding_uu_idx
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:188)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:136)
	at org.adempiere.pipo.srv.PipoDictionaryService.merge(PipoDictionaryService.java:104)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:198)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:166)
	at org.adempiere.plugin.utils.Incremental2PackActivator.installPackage(Incremental2PackActivator.java:120)
	at org.adempiere.plugin.utils.Incremental2PackActivator.access$4(Incremental2PackActivator.java:89)
	at org.adempiere.plugin.utils.Incremental2PackActivator$2.run(Incremental2PackActivator.java:259)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
caused by: org.adempiere.pipo2.exception.DatabaseAccessException: Failed to validate AD_TableIndex for lco_invoicewithholding_uu_idx
	at org.adempiere.pipo2.handler.TableIndexElementHandler.endElement(TableIndexElementHandler.java:99)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:266)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:258)
	at org.adempiere.pipo2.PackInHandler.endElement(PackInHandler.java:334)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.endElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEndElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse(Unknown Source)
	at javax.xml.parsers.SAXParser.parse(Unknown Source)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:173)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:136)
	at org.adempiere.pipo.srv.PipoDictionaryService.merge(PipoDictionaryService.java:104)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:198)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:166)

20:47:27.034===========> Incremental2PackActivator.packIn: Pack in failed. [41]
java.lang.RuntimeException: Failed to validate AD_TableIndex for lco_invoicewithholding_uu_idx
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:188)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:136)
	at org.adempiere.pipo.srv.PipoDictionaryService.merge(PipoDictionaryService.java:104)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:198)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:166)
	at org.adempiere.plugin.utils.Incremental2PackActivator.installPackage(Incremental2PackActivator.java:120)
	at org.adempiere.plugin.utils.Incremental2PackActivator.access$4(Incremental2PackActivator.java:89)
	at org.adempiere.plugin.utils.Incremental2PackActivator$2.run(Incremental2PackActivator.java:259)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$201(Unknown Source)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
caused by: org.adempiere.pipo2.exception.DatabaseAccessException: Failed to validate AD_TableIndex for lco_invoicewithholding_uu_idx
	at org.adempiere.pipo2.handler.TableIndexElementHandler.endElement(TableIndexElementHandler.java:99)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:266)
	at org.adempiere.pipo2.PackInHandler.processElement(PackInHandler.java:258)
	at org.adempiere.pipo2.PackInHandler.endElement(PackInHandler.java:334)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.endElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanEndElement(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(Unknown Source)
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(Unknown Source)
	at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse(Unknown Source)
	at javax.xml.parsers.SAXParser.parse(Unknown Source)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:173)
	at org.adempiere.pipo2.PackIn.importXML(PackIn.java:136)
	at org.adempiere.pipo.srv.PipoDictionaryService.merge(PipoDictionaryService.java:104)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:198)
	at org.adempiere.plugin.utils.Incremental2PackActivator.packIn(Incremental2PackActivator.java:166)

21:00:11.345-----------> ThreadPoolAsynchronousRunner.log: com.mchange.v2.async.ThreadPoolAsynchronousRunner$DeadlockDetector@262c84f0 -- APPARENT DEADLOCK!!! Creating emergency threads for unassigned pending tasks! [43]
21:00:11.347-----------> ThreadPoolAsynchronousRunner.log: com.mchange.v2.async.ThreadPoolAsynchronousRunner$DeadlockDetector@262c84f0 -- APPARENT DEADLOCK!!! Complete Status: 
	Managed Threads: 3
	Active Threads: 3
	Active Tasks: 
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@11929d8
			on thread: C3P0PooledConnectionPoolManager[identityToken->1b608d59o1i7hlrc1654wk7|7273d53, dataSourceName->iDempiereDS]-HelperThread-#1
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@1b0a2478
			on thread: C3P0PooledConnectionPoolManager[identityToken->1b608d59o1i7hlrc1654wk7|7273d53, dataSourceName->iDempiereDS]-HelperThread-#0
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@43b6586b
			on thread: C3P0PooledConnectionPoolManager[identityToken->1b608d59o1i7hlrc1654wk7|7273d53, dataSourceName->iDempiereDS]-HelperThread-#2
	Pending Tasks: 
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask@7866fb87
		com.mchange.v2.resourcepool.BasicResourcePool$1RefurbishCheckinResourceTask@17fdd3bc
		com.mchange.v2.resourcepool.BasicResourcePool$1RefurbishCheckinResourceTask@165b15f7
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@3bf03b17
		com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@18b781bf
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@60e30e14
		com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@5ab174bd
		com.mchange.v2.resourcepool.BasicResourcePool$1DestroyResourceTask@947081c
		com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask@16656b06
Pool thread stack traces:
	Thread[C3P0PooledConnectionPoolManager[identityToken->1b608d59o1i7hlrc1654wk7|7273d53, dataSourceName->iDempiereDS]-HelperThread-#0,5,main]
		java.net.SocketInputStream.socketRead0(Native Method)
		java.net.SocketInputStream.read(Unknown Source)
		java.net.SocketInputStream.read(Unknown Source)
		org.postgresql.core.VisibleBufferedInputStream.readMore(VisibleBufferedInputStream.java:143)
		org.postgresql.core.VisibleBufferedInputStream.ensureBytes(VisibleBufferedInputStream.java:112)
		org.postgresql.core.VisibleBufferedInputStream.read(VisibleBufferedInputStream.java:71)
		org.postgresql.core.PGStream.ReceiveChar(PGStream.java:282)
		org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1718)
		org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
		org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:481)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:361)
		com.mchange.v2.c3p0.impl.NewProxyStatement.executeQuery(NewProxyStatement.java:327)
		com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:262)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:509)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:451)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:443)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:433)
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:2196)
		com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
	Thread[C3P0PooledConnectionPoolManager[identityToken->1b608d59o1i7hlrc1654wk7|7273d53, dataSourceName->iDempiereDS]-HelperThread-#2,5,main]
		java.net.SocketInputStream.socketRead0(Native Method)
		java.net.SocketInputStream.read(Unknown Source)
		java.net.SocketInputStream.read(Unknown Source)
		org.postgresql.core.VisibleBufferedInputStream.readMore(VisibleBufferedInputStream.java:143)
		org.postgresql.core.VisibleBufferedInputStream.ensureBytes(VisibleBufferedInputStream.java:112)
		org.postgresql.core.VisibleBufferedInputStream.read(VisibleBufferedInputStream.java:71)
		org.postgresql.core.PGStream.ReceiveChar(PGStream.java:282)
		org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1718)
		org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
		org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:481)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:361)
		com.mchange.v2.c3p0.impl.NewProxyStatement.executeQuery(NewProxyStatement.java:327)
		com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:262)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:509)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:451)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:443)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:433)
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:2196)
		com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
	Thread[C3P0PooledConnectionPoolManager[identityToken->1b608d59o1i7hlrc1654wk7|7273d53, dataSourceName->iDempiereDS]-HelperThread-#1,5,main]
		java.net.SocketInputStream.socketRead0(Native Method)
		java.net.SocketInputStream.read(Unknown Source)
		java.net.SocketInputStream.read(Unknown Source)
		org.postgresql.core.VisibleBufferedInputStream.readMore(VisibleBufferedInputStream.java:143)
		org.postgresql.core.VisibleBufferedInputStream.ensureBytes(VisibleBufferedInputStream.java:112)
		org.postgresql.core.VisibleBufferedInputStream.read(VisibleBufferedInputStream.java:71)
		org.postgresql.core.PGStream.ReceiveChar(PGStream.java:282)
		org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:1718)
		org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:173)
		org.postgresql.jdbc2.AbstractJdbc2Statement.execute(AbstractJdbc2Statement.java:645)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeWithFlags(AbstractJdbc2Statement.java:481)
		org.postgresql.jdbc2.AbstractJdbc2Statement.executeQuery(AbstractJdbc2Statement.java:361)
		com.mchange.v2.c3p0.impl.NewProxyStatement.executeQuery(NewProxyStatement.java:327)
		com.mchange.v2.c3p0.impl.DefaultConnectionTester.activeCheckConnection(DefaultConnectionTester.java:262)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.testPooledConnection(C3P0PooledConnectionPool.java:509)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:451)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.finerLoggingTestPooledConnection(C3P0PooledConnectionPool.java:443)
		com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.refurbishIdleResource(C3P0PooledConnectionPool.java:433)
		com.mchange.v2.resourcepool.BasicResourcePool$AsyncTestIdleResourceTask.run(BasicResourcePool.java:2196)
		com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
 [43]
